src/test/java/LocationsUtilityTest.java

```diff
@@ -118,4 +119,24 @@ public void getLocationAllTasks() {
         ImmutableList.of("Google Kitchener", "Google Kitchener", ""),
         LocationsUtility.getLocations(PREFIX, ALL_TASKS));
   }
+
+  @Test
```

Comment:
What happens in the null case (i.e. you pass an empty list?). Looks like there should be no issues, but it might be worth adding a test for as this is easy to mess up later

Reply:
Good point, edge case tests are important either way. Thanks!

Reply:
`RESOLVED`